src/components/questions/MultipleChoiceList.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import RadioButtonRN from 'radio-buttons-react-native';
+import { QuestionOption } from '../../models/QuestionOption';
+import FONTS from '../../assets/fonts';
+import COLORS from '../../assets/colors';
+
+interface RadioButtonData {
+  label: string;
+}
+
+const MasterContainer = (props: {
+  questionOption: QuestionOption;
+  onClicked: (label: string) => void;
+}) => {
+  const multipleChoiceQuestions = props.questionOption.extraData as unknown as string[];
+
+  return (
+    <RadioButtonRN
+      data={stringArrayToRadioButtonData(multipleChoiceQuestions)}
+      selectedBtn={(selected: RadioButtonData) => {
+        props.onClicked(selected.label);
+      }}
+      animationTypes={['zoomIn', 'rotate']}
+      textStyle={styles.text}
+      style={styles.radioButton}
+      boxStyle={styles.box}
+      activeColor={COLORS.green}
+      icon={<Icon name="check-circle" size={25} color={COLORS.green} />}
+    />
+  );
+};
+
+function stringArrayToRadioButtonData(stringArray: string[]): RadioButtonData[] {
+  return stringArray.map((string) => {
+    return {
+      label: string,
+    };
+  });
+}
+
+const styles = StyleSheet.create({
+  box: {
+    backgroundColor: COLORS.darkBlue,
```

Comment:
Ik zie toch liever de achtergrond nog steeds wit want het is geen knop en knoppen hebben een achtergrondkleur. Ik denk dat het beter is om of de lijntjes gewoon zwart te doen of die donkerblauw.

Reply:
Heb t aangepast